docs/openapi.yaml

```diff
@@ -794,12 +803,43 @@ paths:
           required: false
           schema:
             type: integer
+      x-codegen-request-body-name: key
+      requestBody:
```

Comment:
A POST body generally seems like the right shape for this endpoint, however, it won't work with the up-coming HTTP caching changes, because the browser/CDN caching is keyed off the URL. 

I wonder if it would be reasonable to specify the ABI as a query param. The sample NFT trait comes out to around 1KB in length after doing `encodeURIComponent(JSON.stringify(traitAbiJson)).length)`. The default max http header size of of nginx, apache, nodejs all appear to be around 8KB to 16KB. Perhaps that's a reasonable limit anyway? We need to impose some size limit on the JSON body anyway so we aren't sending megabytes of sql query to postgres.

Reply:
done!